src/platform.ts

```diff
@@ -232,11 +233,13 @@ export class PowerShellExeFinder {
     private *enumerateAdditionalPowerShellInstallations(): Iterable<IPossiblePowerShellExe> {
         for (const versionName in this.additionalPowerShellExes) {
             if (Object.prototype.hasOwnProperty.call(this.additionalPowerShellExes, versionName)) {
-                const exePath = utils.stripQuotePair(this.additionalPowerShellExes[versionName]);
+                let exePath = utils.stripQuotePair(this.additionalPowerShellExes[versionName]);
                 if (!exePath) {
                     continue;
                 }
 
+                exePath = untildify(exePath);
```

Comment:
@JustinGrote toss a `path.resolve()` around this?

Reply:
Assuming it's handled later on, the only reason I can think not is if you *wanted* the relative path later on if it happened to show up that way. I'm not sure how safe `untildify` is or if it maybe even calls resolve internally.

Reply:
This is the entirety of `untildify`:

```typescript
import os from 'node:os';

const homeDirectory = os.homedir();

export default function untildify(pathWithTilde) {
	if (typeof pathWithTilde !== 'string') {
		throw new TypeError(`Expected a string, got ${typeof pathWithTilde}`);
	}

	return homeDirectory ? pathWithTilde.replace(/^~(?=$|\/|\\)/, homeDirectory) : pathWithTilde;
}
```

I think I just need to test if it's ok here with `..` in the middle of a path.

No path resolution, just substitution for `~`. So like, the VS Code Terminal API is what's picky about having an absolute path for startup for its `cwd` argument.

Reply:
Yeah I think it's fine. `cwd` actually also doesn't need `path.resolve()`...it just needed to be smarter.